content/sensu-go/5.15/reference/checks.md

```diff
@@ -586,6 +586,7 @@ description  | If check requests are published for the check.
 required     | false
 default      | `false`
 type         | Boolean
+default      | `false`
```

Comment:
I already see a default field here on line 587.

Also another odd thing I noticed between 5.15 and 5.18, the lines that type and default live on are reversed.

Reply:
Thank you -- I should have removed that line.

Reply:
I'm not sure I see what you mean by "between 5.15 and 5.18, the lines that type and default live on are reversed." I am probably overlooking it -- can you give an example?

Reply:
5.15:

<img width="859" alt="5 15" src="https://user-images.githubusercontent.com/19964713/75814855-521e4c80-5d47-11ea-9f32-81b45538c7fc.png">

5.18:

<img width="870" alt="5 18" src="https://user-images.githubusercontent.com/19964713/75814871-577b9700-5d47-11ea-9881-0fe5ee57b5c0.png">

Reply:
Thank you! I do not know how I missed that. I probably standardized the order when I copyedited everything.